README.md

```diff
@@ -203,6 +205,7 @@ Shows the below help
             nodeshift resource        resource command
             nodeshift apply-resource  apply resource command
             nodeshift undeploy        undeploy resources
+            nodeshift watch           watch deployment
```

Comment:
@lholmquist does the watch flag cause only deployments to be watched? Or everything? I think it would be good to be a little more descriptive here about what the behavior really is?

Reply:
The plan is for only things that are deployed.  

ATM, if you want to deploy then start watching you need to do that in 2 steps(which is easy with a npm script).  We could add something that combined "deploy" with "watch",  but i would do that in another commit also

Reply:
actually, the real answer is, watches a currently running Pod